### PR TITLE
Removes much of the HipChat implementation.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: lib/cdo/hip_chat.rb [hunk 1]
```diff
@@ -4,31 +4,15 @@
...
-
   TRUNCATION_PREFIX = '[TRUNCATED]'
-  MAX_MESSAGE_SIZE = 10000
+  MAX_MESSAGE_SIZE = 10_000
 
   def self.developers(message, options={})
-    # temporarily redirect developer logging to 'Server operations'.
-    # TODO(dave): rename or split HipChat.developers once we settle on a HipChat logging strategy.
+    # Temporarily redirect developer logging to 'Server operations'.
+    # TODO(dave): rename or split HipChat.developers once we settle on a HipChat
```
Comment (breville) [inline lib/cdo/hip_chat.rb:14]: still HipChat?

File: lib/cdo/hip_chat.rb [hunk 2]
```diff
@@ -57,105 +41,19 @@ def self.slackify(message)
...
 
-  # If CDO.hip_chat_logging is true, post message to hipchat with
-  # exponential backoff, retrying at most `max_retries` times.
-  # Otherwise log to CDO.log.
-  #
-  # Implementation notes: The synchronous post with exponential
-  # backoff is done on a newly spawned thread to avoid blocking the
-  # main Ruby thread.  This is OK for current usage because we log to
-  # Hipchat infrequently.
   def self.post_to_hipchat(room, message, options={})
```
Comment (breville) [inline lib/cdo/hip_chat.rb:44]: worth a rename to generic chat?

**Comment by breville** [comment]
lgtm, but curious whether we should rename to a more generic chat..

**Comment by ashercodeorg** [comment]
Yes, I do intend follow-up PRs to change the name and further clean stuff up.

**Pull request merged**